lib/ddtrace/contrib/rails/framework.rb

```diff
@@ -47,6 +48,10 @@ def self.configure(config)
             port: datadog_config[:trace_agent_port]
           )
 
+          # set default tracer tags
+          datadog_config[:tracer].set_tags('env' => datadog_config[:env])
+          datadog_config[:tracer].set_tags(datadog_config[:tags])
```

Comment:
that was a temporary order before choosing if `env` should be removed or not. In that case we can enforce the `env` tag so that it takes precedence among global tags.